roles/httpd/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+httpd_ssl_dir: /etc/pki/httpd
+httpd_pulp_api_backend: http://localhost:8080
+httpd_pulp_content_backend: http://localhost:8080
+httpd_foreman_backend: http://localhost:3000
```

Comment:
I read a little bit, but seemed that Unix sockets were unfriendly in his container environment with podman and definitely do not extend to Openshift. Is that your read on it as well?

Reply:
That's also my thinking here, yes.

Reply:
We have proxy issues (HTTP 502 IIRC) when we used HTTP that went away when we switched to UNIX sockets. We also saw those issues with proxies from content proxies that went away with HTTP/2. Does Puma support HTTP/2 or how can we avoid introducing those issues again?

Also, you can actually use socket activation with containers where systemd opens the socket on the host and passes the file descriptor to a container. I use that with gunicorn (on something that isn't Pulp) and works well for me. Should we investigate that?

Reply:
Moved to https://github.com/ehelms/foreman-quadlet/issues/31 to allow continued discussion